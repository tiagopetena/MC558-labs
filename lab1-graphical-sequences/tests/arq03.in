15 
10 9 9 9 8 8 8 7 6 6 6 6 6 3 3