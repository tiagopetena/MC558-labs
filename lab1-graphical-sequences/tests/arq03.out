2 3 4 5 6 7 8 9 10 11
1 3 4 5 6 7 8 12 13
1 2 4 5 6 7 8 12 13
1 2 3 5 6 7 9 10 11
1 2 3 4 6 7 9 10
1 2 3 4 5 7 11 13
1 2 3 4 5 6 8 12
1 2 3 7 11 12 13
1 4 5 10 14 15
1 4 5 9 14 15
1 4 6 8 12 13
2 3 7 8 11 13
2 3 6 8 11 12
9 10 15
9 10 14
